source/D2Game/src/UNIT/SUnitDmg.cpp

```diff
@@ -1290,7 +1290,7 @@ void __fastcall SUNITDMG_ExecuteEvents(D2GameStrc* pGame, D2UnitStrc* pAttacker,
 		STATLIST_SetUnitStat(pDefender, STAT_HITPOINTS, nNewHp, 0);
 	}
 
-	if (pDamage->dwDmgTotal > 0)
```

Comment:
I think we should change `dwDmgTotal` to be signed instead of fixing it only here.
For example it appears in `D2Game.0x6FD049D0` too, and it also checks for negative values.